### PR TITLE
Add TRAVIS_DEBIAN_COMPONENTS option

### PR DESCRIPTION
**Pull request by didrocks**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -159,6 +159,14 @@ <h3 id="configuration">
         Which distribution of Debian the package should be built in. By
         default, this is automatically detected from the branch name.
       </dd>
+      <dt>
+        TRAVIS_DEBIAN_COMPONENTS
+      </dt>
+      <dd>
+        Additional components used for building and testing this package.
+        The components are set in sources.list and should be space-separated.
+        By default, this is set to "main".
```
Comment (lamby) [inline index.html]: Please wrap main in tt tag

File: index.html [hunk 2]
```diff
@@ -159,6 +159,14 @@ <h3 id="configuration">
         Which distribution of Debian the package should be built in. By
         default, this is automatically detected from the branch name.
       </dd>
+      <dt>
+        TRAVIS_DEBIAN_COMPONENTS
+      </dt>
+      <dd>
+        Additional components used for building and testing this package.
```
Comment (lamby) [inline index.html]: "Additional archive components" might be useful here.

**Review by lamby — CHANGES_REQUESTED** [review]

**Comment by didrocks** [comment]
Thanks for the review! Both comments addressed

**Comment by didrocks** [comment]
Was merged with PR #56, closing.

**Pull request closed without merging**